.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Nit: Add a newline

Reply:
Whoops, I meant to do this before. Done.